### PR TITLE
Fixed the link to the JSON RPC interface

### PR DESCRIPTION
**Pull request by qbzzt**
The link in the page led to a 404 error instead of https://ethereum.org/en/developers/docs/apis/json-rpc/

## Related Issue

#2222

### CODE REVIEW

File: src/content/developers/docs/programming-languages/index.md [hunk 1]
```diff
@@ -29,5 +29,5 @@ Select your language of choice to find projects, resources, and virtual communit
 If you want to link to resources or point to a virtual community for an additional programming language you can request a new page by [opening an issue](https://github.com/ethereum/ethereum-org-website/issues/new/choose).
 
 If you just want to write code to interface with the blockchain using a currently unsupported language
-you can use the [JSON-RPC interface](/developers/docs/apis/json-rpc/endpoints/) to connect to the Ethereum network. Any programming 
+you can use the [JSON-RPC interface](/en/developers/docs/apis/json-rpc/) to connect to the Ethereum network. Any programming
```
Comment (jley81) [inline src/content/developers/docs/programming-languages/index.md:32]: Well done

**Review by jley81 — APPROVED** [review]

**Review by wackerow — CHANGES_REQUESTED** [review]
Nice, thanks for updating this. Just don't need the `/en` at the beginning. We can just use `/developers/docs/apis/json-rpc/` which will follow existing intl logic.

File: src/content/developers/docs/programming-languages/index.md (hunk 1 above)
Comment (samajammin) [inline src/content/developers/docs/programming-languages/index.md:32]:
```suggestion
you can use the [JSON-RPC interface](/developers/docs/apis/json-rpc/) to connect to the Ethereum network. Any programming 
```

**Review by samajammin — APPROVED** [review]
Thanks!

**Pull request merged**